### PR TITLE
feat: add Action TransportActivityData

### PR DESCRIPTION
**Pull request by raimundo-henriques**

### CODE REVIEW

File: specs/index.bs [hunk 1]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+
+## Action TransportActivityData ## {#action-tad}
+
+Lists the [=data owner=]'s [=Transport Activity Data=] with [[#pagination]] and optional [[#filtering]]
+
+[=Host systems=] SHOULD implement an access management system and only return the [=Transport Activity Data=] for which the [=data owner=] granted access to the requesting [=data recipient=].
+
+### Filtering ### {#filtering}
+
+Filtering CAN be requested by a [=data recipient=] by setting a conforming [=Filter=] request parameter.
```
Comment (zeitgeist) [inline specs/index.bs:1529]: Filtering enables a [=data recipient=] to retrieve a subset of [=Transport Activity Data=]. Filtering is an OPTIONAL capability of a [=host system=] which a data recipent can trigger by setting a [=Filter=] request parameter.

File: specs/index.bs [hunk 2]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+
+Note: The filter statement syntax is described at the [=Filter=] request parameter.
+
+Support for [=Filter|filtering=] by a [=host system=] is OPTIONAL such that
```
Comment (zeitgeist) [inline specs/index.bs:1533]: I'll push a proposal how to formulate this through a separate branch, @raimundo-henriques

File: specs/index.bs [hunk 3]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+<l>[=TadResponseBody=]</l>
+</pre>
+
+With response parameters
+
+: <dfn>TadStatusCode</dfn>
+::
+    If the host system returns transport activity data, the `HttpStatusCode` MUST be either 200 or 202:
+        - `HttpStatusCode` 200 indicates the returned data is the complete result of the given query.
+        - `HttpStatusCode` 202 indicates the returned data is an incomplete result of the given query. The host system MAY return this `HttpStatusCode` if it principally decides that it's able to obtain the remaining data in the future. This `HttpStatusCode` MUST NOT be returned if the request parameter `Filter` is not defined. The [=data recipient=] MAY continue to send the same request with exponential-backoff until it receives the complete result, indicated by `HttpStatusCode` 200.
```
Comment (zeitgeist) [inline specs/index.bs:1624]: let's remove this status code completely, please

File: specs/index.bs [hunk 4]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+
+    If the host system responds with an error response, the `HttpStatusCode` MUST match the HTTP Status Code of the respective error response code, as defined in the [Pathfinder Technical Specifications](https://wbcsd.github.io/tr/2023/data-exchange-protocol-20231207/#api-error-responsesm).
+
+    If the host system does not return the transport activity data, it MUST return an error HTTP Status Code (4xx, 5xx).
```
Comment (zeitgeist) [inline specs/index.bs:1628]: Do we need this additional definition here?

File: specs/index.bs [hunk 5]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+
+: <dfn>TadStatusText</dfn>
+::
+    The HTTP Status text conforming to the HTTP status code [=TadStatusCode=].
+
+: <dfn>TadResponseBody</dfn>
+::
+    If the host system accepts the [=access token=], the body MUST be a JSON object with property `data` with value the list of [=Transport Activity Data=].
+    The list MUST be encoded as a JSON array. If the list is empty, the host system MUST return an empty JSON array.
```
Comment (zeitgeist) [inline specs/index.bs:1637]: Let's remove this sentence, please

File: specs/index.bs [hunk 6]
```diff
@@ -1496,22 +1504,156 @@ The following properties are defined for data type <dfn element>Feedstock</dfn>:
...
+
+    If the request parameter `Filter` is defined, the specified expression SHOULD be evaluated for each Transport Activity Data in the collection, and only [=Transport Activity Data=] where the expression evaluates to true SHOULD be included in the response, as described in [[#filtering]]. ProductFootprints for which the expression evaluates to false or which are not made available for the [=data recipient=] SHOULD be omitted from the list returned in the response.
```
Comment (zeitgeist) [inline specs/index.bs:1639]: This specification / explanation here should be part of the filtering section above.

Comment (zeitgeist) [reply specs/index.bs:1639]: or omitted outright :)

**Review by zeitgeist — CHANGES_REQUESTED** [review]
Hello Raimundo, thank you. Looks good except for some minor editing work.

File: specs/index.bs (hunk 3 above)
Comment (raimundo-henriques) [reply specs/index.bs:1640]: I wasn't entirely sure as to whether you meant the status code 202 or the whole definition, so I decided to go with the former

File: specs/index.bs (hunk 1 above)
Comment (raimundo-henriques) [reply specs/index.bs]: I took this to have been replaced by your PR

**Review by zeitgeist — APPROVED** [review]
LGTM. We might need to update the response documentation on authentication token handling

**Pull request merged**